app/main.dev.ts

```diff
@@ -40,31 +47,23 @@ if (process.env.NODE_ENV === 'production') {
   sourceMapSupport.install();
 }
 
-if (process.env.NODE_ENV === 'development' || process.env.DEBUG_PROD === 'true') {
-  require('electron-debug')();
-}
-
 app.setPath('userData', getUserDataPath(app));
 app.setPath('logs', path.join(getUserDataPath(app), 'logs'));
 
-const installExtensions = () => {
-  const installer = require('electron-devtools-installer');
-  const forceDownload = !!process.env.UPGRADE_EXTENSIONS;
-  const extensions = ['REACT_DEVELOPER_TOOLS', 'REDUX_DEVTOOLS'];
-
-  return Promise.all(
-    extensions.map(name => installer.default(installer[name], forceDownload))
-  ).catch(console.log);
-};
-
 const createWindow = async () => {
   if (process.env.NODE_ENV === 'development' || process.env.DEBUG_PROD === 'true') {
-    await installExtensions();
+    await app.whenReady();
+    // await installExtension([REDUX_DEVTOOLS, REACT_DEVELOPER_TOOLS], {
+    //   loadExtensionOptions: { allowFileAccess: true },
+    //   forceDownload: true,
+    // } as any)
+    //   .then(name => console.log(`Added Extension:  ${name}`))
+    //   // .catch(err => console.log('An error occurred: ', err))
+    //   .finally(() => require('electron-debug')());
```

Comment:
remove

Reply:
This is a frustrating issue. To load in redux devtools you need to add this, but it throws some runtime errors in dev mode.

Reply:
Tidied up with comment and issue ref